derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "extism-pdk-derive"
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
we need this bump here or no?

Reply:
i don't think so, since it doesnt actually depend on extism-manifest